dev-util/weka/weka-3.8.6-r1.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source"
+
+inherit desktop java-pkg-2 java-pkg-simple xdg
+
+DESCRIPTION="A Java data mining package"
+HOMEPAGE="https://ml.cms.waikato.ac.nz/weka"
+SRC_URI="https://downloads.sourceforge.net/project/weka/weka-3-8/${PV}/weka-${PV//./-}.zip"
+S="${WORKDIR}/${P//./-}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+BDEPEND="app-arch/unzip"
+CP_DEPEND="dev-java/javacup:0"
+DEPEND="${CP_DEPEND}
+	virtual/jdk:1.8"
+RDEPEND="${CP_DEPEND}
+	>=virtual/jre-1.8:*"
+
+JAVA_GENTOO_CLASSPATH_EXTRA="lib/bounce.jar:lib/mtj.jar:lib/jfilechooser-bookmarks-0.1.6.jar"
+JAVA_MAIN_CLASS="weka.gui.GUIChooser"
+JAVA_RESOURCE_DIRS="src/main/res"
+JAVA_SRC_DIR="src/main/java"
+
+src_prepare() {
+	java-pkg-2_src_prepare
+	unzip -qq "${PN}-src.jar" -d . || die "Failed to unpack the source"
+	java-pkg_clean \
+		! -path ./lib/arpack_combined.jar \
+		! -path ./lib/bounce.jar \
+		! -path ./lib/commons-compress-1.21.jar \
+		! -path ./lib/core.jar \
+		! -path ./lib/flatlaf-2.0.jar \
+		! -path ./lib/hamcrest-core-1.3.jar \
+		! -path ./lib/istack-commons-runtime-3.0.12.jar \
+		! -path ./lib/jakarta.activation-api-1.2.2.jar \
+		! -path ./lib/jakarta.xml.bind-api-2.3.3.jar \
+		! -path ./lib/java-cup-11b-runtime.jar \
+		! -path ./lib/java-cup-11b.jar \
+		! -path ./lib/jaxb-runtime-2.3.5.jar \
+		! -path ./lib/jclipboardhelper-0.1.2.jar \
+		! -path ./lib/jfilechooser-bookmarks-0.1.6.jar \
+		! -path ./lib/jflex-full-1.8.2.jar \
+		! -path ./lib/junit-4.13.2.jar \
+		! -path ./lib/mtj.jar
+
+	# java-pkg-simple wants resources in JAVA_RESOURCE_DIRS.
+	mkdir -p src/main/res || die
+	pushd src/main/java > /dev/null || die
+		find -type f \
+			! -name '*.java' \
+			| xargs cp --parent -t ../res || die
+	popd > /dev/null || die
+}
+
+src_install() {
+	java-pkg-simple_src_install
+	java-pkg_dojar lib/{arpack_combined,bounce,core,java-cup-11b,java-cup-11b-runtime,mtj}.jar
+	java-pkg_dojar lib/commons-compress-1.21.jar
```

Comment:
Please don't bundle what we have in the tree.

Reply:
Sorry didn't notice some of the jar are already in the tree.

Reply:
> Sorry didn't notice some of the jar are already in the tree.

np. if need help pls tell.

Reply:
Well the jflex version on the tree is older that the one provided by weka, but the application works ok.

Other DEPEND use newer versions, since everthing looks ok I didn't lock to any versions.